<launch>
    <group ns="nlink1">
        <node pkg="nlink_parser" type="linktrack" name="linktrack0" output="screen">
            <param name="port_name" value="/dev/ttyUSB1" />
            <param name="baud_rate" value="921600" />
        </node>
        <node pkg="msg_serialize" type="twist_deserialization" name="twist_deserialization" output="screen">
            <remap from="node_frame" to="nlink_linktrack_nodeframe0"/>
        </node>
    </group>
</launch>
